src/core/collection/Collection.tsx

```diff
@@ -299,7 +299,13 @@ export function EventSelectionCollection({
     },
   });
 
-  const eventsList = data?.eventsByIds;
+  // Reduce past events that are no longer available and therefore do not need to be displayed
+  const eventsList = data?.eventsByIds.data.reduce((availableEvents, event) => {
```

Comment:
Reduce seems confusing, isn't filter enough?
`const eventsList = data?.eventsByIds.data.filter(event => !isEventClosed(event));`

Reply:
I would need a combination of a filter and a map functions and TS would still whine about it 😐

Reply:
updated